File: vector/sort.py
import random
import time

def generator(count, max):
	return [random.randint(0, max) for i in range(count)]


def sort(arr):
	n = len(arr)
	for i in range(n):
		for j in range(0, n - i - 1):
			if arr[j] > arr[j + 1]:
				arr[j], arr[j + 1] = arr[ j+ 1], arr[j]
				

arr = generator(10000, 1000)

t1 = time.time()
sort(arr)
t2 = time.time()

print(t2 - t1)

File: vector/sort.js
'use strict';

function generate(count, range) {
	const res = [];
	for (let i = 0; i < count; i++) {
		res.push(Math.random() * range);
	}
	return res;
}

const arr = generate(100000, 1000);

function bubbleSort(arr) {
	let swap;
	for (let i = 0; i < arr.length - 1; i++) {
		for (let j = 0; j < arr.length - i - 1; j++) {
			if (arr[j] > arr[j + 1]) {
				swap = arr[j];
				arr[j] = arr[j + 1];
				arr[j + 1] = swap;
			}
		}
	}
}

console.time('sort');

bubbleSort(arr);

console.timeEnd('sort')